docs/docs/99-api/examples/automation_demo/automation_demo.js

```diff
@@ -8,7 +8,7 @@
  *
 
  */
-const { draw } = require("../../../../../dist/memory_models_rough.js");
+const { draw } = require("../../../../../memory-viz/dist/memory-viz.bundle.js");
```

Comment:
This is a great catch. However since this is intended to be used for the demo website, I think the best approach is actually to change this to just `require("memory-viz")` (i.e., the package name). You should be able to run the demo file either from the root level of the repository (e.g. `node path/to/demo/automation_demo.js`), and possibly even from within the same directory as the demo file itself, which I expect is what you're currently trying.